index.bs

```diff
@@ -3680,31 +3676,33 @@ To <dfn>trigger event-level attribution</dfn> given an [=attribution trigger=] |
     :: null
     : [=attribution rate-limit record/event-level report ID=]
     :: |report|'s [=event-level report/report ID=]
-1. If the result of running [=check if attribution should be blocked by rate limits=]
-    with |trigger|, |sourceToAttribute|, and |rateLimitRecord| is not null, return it.
 1. If |sourceToAttribute|'s [=attribution source/event-level attributable=] value
```

Comment:
Let's move this above the `Let |rateLimitRecord| be...` step since that variable isn't needed yet.

Reply:
This PR is about conditioning this check with `If sourceToAttribute’s number of event-level reports is less than sourceToAttribute’s max number of event-level reports:`

Should we move both? Then what about the "Otherwise?"

Reply:
Prior to this PR, `|rateLimitRecord|` was needed in the step immediately following its creation. Now it is not.

Reply:
I'm confused -- the section highlighted, `...with |trigger|, |sourceToAttribute|, and |rateLimitRecord| is not null...`, uses the variable, `rateLimitRecord`. How can we move it "above the `Let |rateLimitRecord| be...` step" ?

Reply:
[Step 14](https://pr-preview.s3.amazonaws.com/giladbarkan-github/attribution-reporting-api/pull/1326.html#ref-for-attribution-rate-limit-record%E2%91%A0%E2%91%A4) creates it but it isn't needed until Step 17.1. Please move creation of `|rateLimitRecord|` directly above what is currently Step 17.

Reply:
Done!